docs/source/release_notes/release-notes.rst

```diff
@@ -112,7 +112,8 @@ develop
           (`Pull Request <https://github.com/palantir/atlasdb/pull/2535>`__)
 
     *   - |devbreak|
-        - Simplify and annotate the constructors for SerializableTransactionManager. This should make the code of that class more maintainable.
+        - Simplify and annotate the constructors for ``SerializableTransactionManager``. This should make the code of that class more maintainable.
+          If you used one of the deleted constructors, consider using either the static ``create`` method or the canonical constructor.
```

Comment:
nit: 'consider using' seems a bit off, since they _must_ change the constructor. We should also prefer the static create method over the canonical constructor - does the constructor give you anything the create method doesn't?